server/src/test/java/org/opensearch/index/mapper/MapperServiceTests.java

```diff
@@ -187,7 +186,7 @@ public void testPartitionedConstraints() {
             client().admin()
                 .indices()
                 .prepareCreate("test-index")
-                .addMapping("type", "{\"type\":{\"_routing\":{\"required\":true}}}", XContentType.JSON)
+                .setMapping("{\"" + MapperService.SINGLE_MAPPING_NAME + "\":{\"_routing\":{\"required\":true}}}")
```

Comment:
I think this is not correct, we have quite a few variations, for example:

```
.prepareCreate("test2")
.setMapping("{\"properties\":{\"foo\":{\"type\": \"keyword\"},\"bar\":{\"properties\":{\"baz\":{\"type\":\"keyword\"},"
```

It think this is the one we should be using (no MapperService.SINGLE_MAPPING_NAME), no?

Reply:
nice catch! and yes! fixed.